### PR TITLE
Fixes #19664 - fix tests after admin made unavailable in taxonomies

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: test/katello_test_helper.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ module FixtureTestCase
 
     Setting::Content.load_defaults
 
-    @@admin = ::User.find(FIXTURES['users']['admin']['id'])
+    @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
```
Comment (ares) [inline test/katello_test_helper.rb]: I hope I can ignore this :-)

**Comment by dLobatog** [comment]
Seems good to me and tests passed. The commit summary length can be fixed on merge. Thanks @ares

File: test/katello_test_helper.rb (hunk 1 above)
Comment (dLobatog) [inline test/katello_test_helper.rb]: @ares From @johnpmitsch on IRC - "can you add an exception for the class variable so rubocop ignores it?"

Comment (ares) [inline test/katello_test_helper.rb]: added on top of module

Comment (jlsherrill) [inline test/katello_test_helper.rb]: so i think hound is throwing up an error here, we don't enable the class var cop, and now rubocop is failing with: 
`W: Unnecessary disabling of Style/ClassVars.`

in jenkins :( 

It seems like hound doesn't honor .rubocop.yaml files in subdirectories:  https://github.com/Katello/katello/blob/master/test/.rubocop.yml
hound--

Comment (jlsherrill) [inline test/katello_test_helper.rb]: i'm going to open a PR to move this to the top level rubocop.yml so that the behavior is consistent.

Comment (jlsherrill) [inline test/katello_test_helper.rb]: Still failing: 
01:56:22.484 /var/lib/workspace/workspace/test_katello_pull_request/slave/fast/plugin/test/katello_test_helper.rb:65:1: W: Unnecessary disabling of Style/ClassVars.
01:56:22.484 # rubocop:disable Style/ClassVars

Comment (jlsherrill) [inline test/katello_test_helper.rb]: just remove that line, i think hound should now be fixed

**Comment by ares** [comment]
There's still one issue when combined with Foreman PR, it seems that some foreman task test fails, investigating.

**Comment by ares** [comment]
The problem is now on with foreman-tasks, opened https://github.com/theforeman/foreman-tasks/pull/252 but there's not way to run tests in jenkins with PR from core, katello and foreman-tasks at the same time.

I spoke with Ivan, the patch seems alright and the foreman-tasks release is planned for this afternoon, then I can rerun the tests which should confirm that this PR will keep Katello tests running after CVE PR is merged.

**Comment by jlsherrill** [comment]
ACK thanks @ares

**Pull request merged**